background/lib/polyfill.js

```diff
@@ -0,0 +1,54 @@
+window.browserName = null;
+window.firefoxVersion = null;
+
+async function _setBrowserName() {
+  if (typeof browser.runtime.getBrowserInfo === 'undefined') {
```

Comment:
Yes, but at time of writing I didn't found an official way to detect Chrome.
Alternative would be to query the UserAgent, but I guess that would cause more issues. (UserAgent switchers and the parsing is tricky)